portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,31 @@
-#event-end-time-warning {
-  visibility: hidden; 
+.container-fluid {
+  font-family: Calibri, Candara, Segoe, 'Segoe UI', Optima, Arial, sans-serif;
 }
 
-.new-event-card {
-  border-style: solid; 
+.scrollable {
   height: 300px;
-  width: 400px; 
+  overflow: scroll;
+}
+
+#calendar-view-wrapper,
+#new-event-wrapper,
+#new-task-wrapper,
+#schedule-result-wrapper {
+  background-color: #fffaf5;
+  border-radius: 50px;
+  padding: 3%;
+}
+
+#nav-bar {
+  background-color: #dae8fe;
+  width: 100%;
+}
+
+#event-end-time-warning,
+#event-duplicate-warning {
+  visibility: hidden;
```

Comment:
Consider using d-none class name to control visibility, rather than this custom CSS

Reply:
Holly is dealing with this in her pr.